.github/workflows/ci.yml

```diff
@@ -1,6 +1,8 @@
 on:
   pull_request: {}
-  push: {}
+  push:
+    branches:
+      - main
```

Comment:
This isn’t technically relevant to this PR, but I probably should have set it up this way to begin with. (Every test action is currently running twice on each push to a PR right now — once for the push and once for the update to the PR — and this makes them only run once).